nix/lib/default.nix

```diff
@@ -178,6 +178,7 @@ in rec {
           taskclusterProxy: true
         env:
           APP: "${name}"
+          DEPLOYMENT: "${branch}"
```

Comment:
this is actually not needed since taskcluster is only used to build / run tests. ``DEPLOYMENT`` variable will be set in heroku admin interface since it is a run time variable.